bears/yaml/TravisLintBear.py

```diff
@@ -25,6 +28,29 @@ class TravisLintBear:
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
 
+    # IP Address of www.google.com
+    CHECK_CONNECTION_URL = 'http://216.58.218.174'
```

Comment:
They haven't mentioned that specifically anywhere and I'm still searching for it. :smile: